src/frontend/efiling-frontend/src/modules/helpers/CurrencyUtil.js

```diff
@@ -0,0 +1,10 @@
+import Dinero from "dinero.js";
```

Comment:
never heard of this one before ;-)

Reply:
Nope, me neither. That's just what the app was using everywhere, so I refactored it out to a common util method so we can test it